test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1

```diff
@@ -600,6 +602,15 @@ Describe "Hard link and symbolic link tests" -Tags "CI", "RequireAdminOnWindows"
             New-Item -ItemType Junction -Path $junctionToDir -Value $realDir > $null
             Test-Path $junctionToDir | Should -BeTrue
         }
+        It "New-Item -Force can overwrite a junction" -Skip:(-Not $IsWindows){
+            $rd2 = Get-Item -Path $realDir2
+            New-Item -Name testfile.txt -ItemType file -Path $realDir
+            New-Item -ItemType Junction -Path $junctionToDir -Value $realDir > $null
```

Comment:
I wonder why we create the `testfile.txt`?

Reply:
so there are contents of the directory, the error on no force only appears that way.

Reply:
Actually I can't get the error to throw in the test.  Even though I can recreate it locally. Can you see anything wrong? I think the folder structure in my screenshot should be the same as the mock in the test context... 😕
![image](https://user-images.githubusercontent.com/19718175/196626470-190e76e7-e5ff-4a61-8023-980bdbd40e0f.png)

Reply:
Locally you create the file in second folder, in CI test - in first one.

Reply:
![image](https://user-images.githubusercontent.com/19718175/196631236-f635bbae-f247-434d-8760-83461df9c244.png)
I don't think that is right. 
`$junctionToDir` will point to contents of `$realDir` on the first two runs, and testfile.txt is inside $realDir?

Reply:
In the case are you sure you use right build locally?

Reply:
Yes, I downloaded it from the PR artifact here - https://dev.azure.com/powershell/PowerShell/_build/results?buildId=111574&view=artifacts&pathAsName=false&type=publishedArtifacts

Reply:
It is side effect from previous test where the junction was already created. I think we should remove the juction in previous test.

Reply:
Unfortunately that wasn't the issue I think. 
https://dev.azure.com/powershell/PowerShell/_build/results?buildId=111582&view=ms.vss-test-web.build-test-results-tab&runId=1957440&resultId=100074&paneView=debug